src/tracker-component.js

```diff
@@ -4,150 +4,31 @@ import 'whatwg-fetch'; // eslint-disable-line import/no-unassigned-import
-}) => string;
-
-type ParamsToTokenUrl = () => string;
-
-type ParamsToPropertyIdUrl = () => string;
-
-type JetloreConfig = {|
-    user_id : string,
-    cid : string,
-    feed_id : string,
-    div? : string,
-    lang? : string
-|};
-
-type Config = {|
-    user? : {|
-        email? : string, // mandatory if unbranded cart recovery
-        name? : string
-    |},
-    propertyId? : string,
-    paramsToBeaconUrl? : ParamsToBeaconUrl,
-    paramsToTokenUrl? : ParamsToTokenUrl,
-    jetlore? : {|
-        user_id : string,
-        access_token : string,
-        feed_id : string,
-        div? : string,
-        lang? : string
-    |},
-    paramsToPropertyIdUrl? : ParamsToPropertyIdUrl
-|};
-
-const storage = {
-    paypalCrCart: 'paypal-cr-cart',
-    paypalCrCartExpiry: 'paypal-cr-cart-expiry'
-};
-
-const sevenDays = 6.048e+8;
-
-const accessTokenUrl = 'https://www.paypal.com/muse/api/partner-token';
-
-const getUserIdCookie = () : ?string => {
-    return getCookie('paypal-user-id') || null;
-};
-
-const setRandomUserIdCookie = () : void => {
-    const ONE_MONTH_IN_MILLISECONDS = 30 * 24 * 60 * 60 * 1000;
-    setCookie('paypal-user-id', generate.generateId(), ONE_MONTH_IN_MILLISECONDS);
-};
-
-const composeCart = (type, data) => {
-    // Copy the data so we don't modify it outside the scope of this method.
-    let _data = { ...data };
-
-    // Devnote: Checking for cookie for backwards compatibility (the cookie check can be removed
-    // a couple weeks after deploy because any cart cookie storage will be moved to localStorage
-    // in this function).
-    const storedCart = window.localStorage.getItem(storage.paypalCrCart) || getCookie(storage.paypalCrCart) || '{}';
-    const expiry = window.localStorage.getItem(storage.paypalCrCartExpiry);
-    const cart = JSON.parse(storedCart);
-    const currentItems = cart ? cart.items : [];
-
-    if (!expiry) {
-        window.localStorage.setItem(storage.paypalCrCartExpiry, Date.now() + sevenDays);
-    }
-
-    switch (type) {
-    case 'add':
-        _data.items = addToCart(data.items, currentItems);
-        break;
-    case 'set':
-        _data.items = data.items;
-        break;
-    case 'remove':
-        _data = { ...cart, ...data };
-        _data.items = removeFromCart(data.items, currentItems);
-        break;
-    default:
-        throw new Error('invalid cart action');
-    }
-
-    window.localStorage.setItem(storage.paypalCrCart, JSON.stringify(_data));
-
-    return _data;
-};
+import { composeCart } from './lib/compose-cart';
+import { track } from './lib/track';
+import constants from './lib/constants';
+import type {
+    CartEventType,
+    TrackingType,
+    ViewData,
+    CartData,
+    CancelCartData,
+    RemoveCartData,
+    PurchaseData,
+    UserData,
+    IdentityData,
+    JetloreConfig,
+    Config
+} from './lib/types';
+
+const {
+    accessTokenUrl,
+    storage,
+    defaultTrackerConfig
+} = constants;
```

Comment:
why not just `import { accessokenUrl .... } from './lib/constants'`

Reply:
Each key would have to be exported separately in that case.